src/main/java/com/amazon/opendistroforelasticsearch/ad/rest/RestAnomalyDetectorJobAction.java

```diff
@@ -80,14 +83,33 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
         String detectorId = request.param(DETECTOR_ID);
         long seqNo = request.paramAsLong(IF_SEQ_NO, SequenceNumbers.UNASSIGNED_SEQ_NO);
         long primaryTerm = request.paramAsLong(IF_PRIMARY_TERM, SequenceNumbers.UNASSIGNED_PRIMARY_TERM);
+        boolean historical = request.paramAsBoolean("historical", false);
```

Comment:
Can we infer its value based on whether detectionDateRange is null or not when starting a detector and check detector type when stopping a detector? This can help ease of use.

Reply:
We don't have different detector types in unified flow. Before unified flow, we have historical and realtime detector types. But in unified flow we create two types of detections under the same detector: realtime detection and historical detection. 
When we stop detector, we need user to specify which detection task they want to stop.